lib/commands/file-movement.js

```diff
@@ -318,13 +318,16 @@ async function pullFromRealDevice (device, remotePath, isFile) {
 /**
  * Get bundleIds which can mount by `--documents` flag
  *
+ *
  * @param {Object} udid - The udid of the target device
- * @returns {Array<string>} A list of bundle ids
+ * @returns {Array<string>} A list of User level apps' bundle ids which has
+ *                          'UIFileSharingEnabled' attribute.
+ *                          Only user apps might have it.
  */
 async function getAvailableBundleIds (udid) {
   const service = await services.startInstallationProxyService(udid);
   try {
-    const applications = await service.listApplications();
+    const applications = await service.listApplications({applicationType: 'User'});
```

Comment:
I haven't find UIFileSharingEnabled in system apps so far.
We can ignore it, for now, to improve find app performance.

Reply:
Good idea